lib/manual_relocator.rb

```diff
@@ -37,11 +37,11 @@ def fetch_manual(slug)
   end
 
   def old_section_ids
-    @old_section_ids ||= old_manual.editions.flat_map(&:section_ids).uniq
+    @old_section_ids ||= old_manual.editions.flat_map(&:section_uuids).uniq
   end
 
   def new_section_ids
-    @new_section_ids ||= new_manual.editions.flat_map(&:section_ids).uniq
+    @new_section_ids ||= new_manual.editions.flat_map(&:section_uuids).uniq
```

Comment:
Minor: Is it worth renaming `@old_section_ids` & `@new_section_ids` at the same time?